packages/rest/src/providers/parse-params.provider.ts

```diff
@@ -58,9 +59,27 @@ export class ParseParamsMiddlewareProvider implements Provider<Middleware> {
 
   value(): Middleware {
     return async (ctx, next) => {
+      const requestBodyParser = await ctx.get(RestBindings.REQUEST_BODY_PARSER);
+      const validationOptions: ValidationOptions =
+        (await ctx.get(
+          RestBindings.REQUEST_BODY_PARSER_OPTIONS.deepProperty('validation'),
+          {optional: true},
+        )) ?? DEFAULT_AJV_VALIDATION_OPTIONS;
+      const ajvFactory = await ctx.get(RestBindings.AJV_FACTORY, {
+        optional: true,
+      });
+
       const route: ResolvedRoute = await ctx.get(RestBindings.Operation.ROUTE);
       debug('Parsing parameters for %s %s', route.verb, route.path);
-      const params = await this.parseParams(ctx.request, route);
+      const params = await parseOperationArgs(
```

Comment:
Then the `parseParams` in the constructor is not needed any more

Reply:
Good catch. I'll remove it.